.github/workflows/ci.yml

```diff
@@ -63,14 +71,43 @@ jobs:
         run: cmake --preset ci -DPE_USE_VENDORED_Z3=OFF -DLLVM_EXTERNAL_LIT=$(which lit) -DLLVM_Z3_INSTALL_DIR=/usr/local
 
       - name: Build ${{ matrix.build-type }} with sanitizers set ${{ matrix.sanitizers }}
-        run: cmake --build --preset ci --config ${{ matrix.build-type }} -j $(nproc)
+        run: cmake --build --preset ci --config ${{ matrix.build-type }} -j$(nproc)
+      - name: Test ${{ matrix.build-type }} with sanitizers set ${{ matrix.sanitizers }}
+        run: |
+          lit ./builds/ci/test -D BUILD_TYPE=${{ matrix.build-type }} -v -DCI_OUTPUT_FOLDER=${{ github.workspace }}/builds/ci/test/ghidra/Output
```

Comment:
Before running LIT test, we need to build docker image `trailofbits/patchestry-decompilation` locally. I see the tests is failing because the image is unavailable and it is trying to pull from register.